docs/pre.sh

```diff
@@ -1,42 +1,42 @@
 #!/usr/bin/env bash
 
 ## Create modules pages
-for D in ../x/*; do
+for D in x/*; do
   if [ -d "${D}" ]; then
-    MODDOC=docs/build/modules/$(echo $D | awk -F/ '{print $NF}')
+    MODDOC=docs/docs/build/modules/$(echo $D | awk -F/ '{print $NF}')
```

Comment:
did a pr get backported that wasnt suppose to?

Reply:
Ive just looked into it and it doesnt seem like it, otherwise there could be a chance its been broken for a while

Reply:
hmm, now I dont think this will fix the issue in the docs-repo